src/components/Actions.tsx

```diff
@@ -35,6 +36,11 @@ export const SelectedShapeActions = ({
     getNonDeletedElements(elements),
     appState,
   );
+
+  let isSingleElementBoundContainer = false;
+  if (targetElements.length === 2 && hasBoundTextElement(targetElements[0])) {
```

Comment:
this won't work if the text element is first in the array. Probably not possible now, but could be in the future (e.g. when we support selecting a text and container and creating a bound container from it via contextMenu)

Reply:
yeah, that's the reason I didn't check the 2nd element as it's not possible rn. But I can add a check for 2nd just as well so its not broken in future